src/config.c

```diff
@@ -596,23 +596,25 @@ char *locate_auxiliary_file(const char *scope, const char *path, const char *inc
 	// Fall back to searching in user config directory
 	scoped_charp picom_scope = mstrjoin("/picom/", scope);
 	scoped_charp config_home = (char *)xdg_config_home();
-	char *ret = locate_auxiliary_file_at(config_home, picom_scope, path);
-	if (ret) {
-		return ret;
+	if (config_home && strlen(config_home)) {
```

Comment:
do we need to check strlen?

Reply:
hm, it seems that the `xdg_config_home` function can't return an empty string. let's get rid of this check then.